src/main/java/frc/robot/Constants.java

```diff
@@ -21,10 +21,10 @@
  * (log replay from a file).
  */
 public final class Constants {
-  public static final Mode simMode = Mode.SIM;
-  public static final Mode currentMode = RobotBase.isReal() ? Mode.REAL : simMode;
+  public static final Mode currentMode = RobotBase.isReal() ? Mode.REAL : Mode.SIM;
```

Comment:
⛏️ The naming of this with a leading lowercase I don't really like since it's a constant, but I'm not sure what's most common.